src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java

```diff
@@ -205,33 +205,16 @@ public void testAdd() {
     Assert.assertEquals(2, ranges3[1].getValues().length);
     Assert.assertEquals(labels3, ranges3[1].getLables());
 
-    // Failure cases
-    try {
-      client.add("seriesAdd", 800L, 1.1);
-      Assert.fail();
-    } catch(JedisDataException e) {
-      // Error on creating same rule twice
```

Comment:
This comment is false and was copy pasted :(

Reply:
luckily it's not there any more :)